### PR TITLE
Move travis logic into a bash script

### PR DESCRIPTION
**Pull request by fejta**
/assign @clarketm @BenTheElder

### CODE REVIEW

File: presubmit.sh [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  cd $GOPATH/src/k8s.io/repo-infra
+else
+  echo "We do not appear to be in travis CI..." >&2
+  cd "$(git rev-parse --show-toplevel)"
+fi
+
+echo "deb [arch=amd64] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list
+curl https://bazel.build/bazel-release.pub.gpg | sudo apt-key add -
+sudo apt-get update
+sudo apt-get install bazel
```
Comment (BenTheElder) [inline presubmit.sh]: we'll want to drop all this sudo and maybe use bazelisk or a pre-installed bazel later

File: presubmit.sh [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+go get -u github.com/alecthomas/gometalinter
+go get -u github.com/bazelbuild/buildtools/buildifier
+gometalinter --install
```
Comment (BenTheElder) [inline presubmit.sh]: also not related to this PR but gometalinter is dead / archived and points users to https://github.com/golangci/golangci-lint which is pretty great mostly

**Comment by fejta** [comment]
Yeah I actually had made other more aggressive changes in the past, but want to start by just getting this to run on prow, and then will more better things.

**Comment by fejta** [comment]
Can I get another lgtm from one of you? Needed to add a cd in the .travis.yml

**Comment by fejta** [comment]
NM, looks like I'm still not getting this to work on travis

**Pull request merged**